charts/monitoring-config/templates/kube-prometheus-stack/config/oauth2-proxy.tpl

```diff
@@ -0,0 +1,50 @@
+{{- define "monitoring-config.oauth2-proxy.standard-env" -}}
+- name: OAUTH2_PROXY_ALLOWED_GROUP
+  value: {{ .Values.monitoring.authorisation.readWriteGithubTeam }}
+- name: OAUTH2_PROXY_OIDC_ISSUER_URL
+  value: "https://dex.{{ .Values.k8sExternalDomainSuffix }}"
+- name: OAUTH2_PROXY_PROVIDER
+  value: oidc
+- name: OAUTH2_PROXY_PROVIDER_DISPLAY_NAME
+  value: GitHub
+{{- end -}}
+
+{{- define "monitoring-config.oauth2-proxy.alb-annotations" -}}
+alb.ingress.kubernetes.io/scheme: internet-facing
+alb.ingress.kubernetes.io/target-type: ip
+alb.ingress.kubernetes.io/listen-ports: '{"HTTP": 80, "HTTPS": 443}'
+alb.ingress.kubernetes.io/ssl-redirect: "443"
+alb.ingress.kubernetes.io/ssl-policy: ELBSecurityPolicy-TLS-1-2-Ext-2018-06
+{{- end -}}
+
+{{- define "monitoring-config.oauth2-proxy.config" -}}
+proxyVarsAsSecrets: false
+ingress:
+  enabled: true
+  pathType: Prefix
+  hosts: [ "{{ .app }}.{{ .Values.k8sExternalDomainSuffix }}" ]
+  annotations:
+    alb.ingress.kubernetes.io/load-balancer-name: {{ .app }}
+{{ include "monitoring-config.oauth2-proxy.alb-annotations" . | indent 4 }}
+  extraArgs:
+    skip-provider-button: "true"
+  extraEnv:
+{{ include "monitoring-config.oauth2-proxy.standard-env" . | indent 4 }}
+    - name: OAUTH2_PROXY_UPSTREAMS
+      value: "http://kube-prometheus-stack-{{ .app }}:{{ .port }}"
+    - name: OAUTH2_PROXY_CLIENT_ID
+      valueFrom:
+        secretKeyRef:
+          name: govuk-dex-alertmanager
```

Comment:
Is this meant to be `govuk-dex-alertmanager` for the Prom one as well?

Reply:
Nope, I missed that one when I was templating out the variables

Reply:
cool no worries!